Kassiopeia/Interactions/Source/KSIntCalculatorMott.cxx

```diff
@@ -143,14 +143,14 @@ std::vector<double> KSIntCalculatorMott::RMott_coeffs(double const E0) {
     std::vector<double> a(6, 0.0); // Initialize a with 6 zeros, the last entry is not related to coefficient calculation it is Z for the nucleus of choice
     std::vector<std::vector<double>> b(5, std::vector<double>(6));
 
-    if (fNucleus == "He") {
+    if (fNucleus == "He-4") { // independent of mass number, dependent on atomic number (ie. He-3 would have same data as He-4)
```

Comment:
How about instead of the comment just

```
#include "KStringUtils.h"
```

at the top and then 
```
if (KStringUtils::IStartsWith(fNucleus, "He-") {
```
? ``IStartsWith is the case insensitive check, if a string starts with another string. But this works also as it is now of course.

Reply:
(edit: I missed a closing bracket in an earlier version of that suggestion)

Reply:
This works well, I like it better than the comment

Reply:
I think we just do it if you have the time to implement it, you've already done much here, so I wouldn't force it from my side. I also prefer this solution though.

Reply:
Updated

Reply:
Reverted back to original, not sure why it failed

Reply:
Ah, it failed since `KStringUtils` is in the namespace `katrin`, so you can either use it by declaring the namespace in advance like `katrin::KStringUtils` or - since you are in a source file and not a header file, where that would be frowned upon - you can use `using namespace katrin;` after the `include` statements so it finds the namespace automatically. Btw. for completeness: You could also just do `using katrin::KStringUtils` to just import that symbol from the namespace. I have to admit I didn't test the code before suggesting it, so that's partly on me.

Reply:
Yes, that did the trick. Thank you!